setup.py

```diff
@@ -1,10 +1,34 @@
 import setuptools
+from pkg_resources import DistributionNotFound, get_distribution
 
 from tinytuya import __version__
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+INSTALL_REQUIRES = [
+    'requests',      # Used for Setup Wizard - Tuya IoT Platform calls
+    'colorama',      # Makes ANSI escape character sequences work under MS Windows.
+]
+
+CHOOSE_CRYPTO_LIB = [
+    'cryptography',  # pyca/cryptography - https://cryptography.io/en/latest/

```

Comment:
I like this. It has good review, works well in my tests and is often listed above pycryptodome in ratings. However, it does not support Python 2.7.  I suggest we officially remove 2.7 support with this PR.

Reply:
Sounds good to me.  My computer died last month and I installed Debian 12 on my new one, so I can't even test against Python 2.7 anymore without copying it to another machine first.